src/DiscordServer.js

```diff
@@ -213,7 +213,7 @@ class DiscordServer {
       Cache.set(`bindings.${userid}`, '__groups', groups)
```

Comment:
Should this not be storing `groups.data` if `groups.data` is being returned?

Reply:
I wasn't exactly sure what to do with that. I left it as is because I only saw something being pulled out of cache for that 1 time, which was as the groups variable. I could be wrong.

Reply:
This cache is designed to store the group data so it can be iterated upon straight from cache, designed to return the same value from the function as if the function was called fresh (Please add the `.data`)

Reply:
Resolved with review fixes.